variables.tf

```diff
@@ -66,3 +66,21 @@ variable "metadata" {
   description = "Metadata to attach to the instance."
   default     = {}
 }
+
+variable "admin" {
```

Comment:
Let's insert some reference for users of this module, regarding how to pass secrets without tracking them VSC.  [This](https://developer.hashicorp.com/terraform/tutorials/configuration-language/sensitive-variables#set-values-with-a-tfvars-file) for example.